experimental/zerotrie/src/zerotrie.rs

```diff
@@ -204,6 +204,24 @@ macro_rules! impl_zerotrie_subtype {
             pub fn is_empty(&self) -> bool {
                 self.store.as_ref().is_empty()
             }
+            /// Gets the value at the head of the trie. This is equivalent to
+            /// calling `get` with the empty string.
+            ///
+            /// # Examples
+            ///
+            /// ```
+            #[doc = concat!("use zerotrie::", stringify!($name), ";")]
+            ///
+            /// // A trie with two values: "" and "abc"
+            #[doc = concat!("let trie: &", stringify!($name), "<[u8]> = ", stringify!($name), "::from_bytes(b\"\\x80abc\\x81\");")]
+            ///
+            /// assert_eq!(Some(0), trie.head_value());
+            /// assert_eq!(Some(0), trie.get(""));
+            /// ```
+            #[inline]
+            pub fn head_value(&self) -> Option<usize> {
```

Comment:
Please bikeshed the name of this function.

Reply:
it can be just `head`